src/package.xml

```diff
@@ -960,6 +952,26 @@
         <members>Partial_Soft_Credit__c.Contact__c</members>
         <members>Partial_Soft_Credit__c.Opportunity__c</members>
         <members>Partial_Soft_Credit__c.Role_Name__c</members>
+        <members>Schedulable__c.Active__c</members>
+        <members>Schedulable__c.Class_Name__c</members>
+        <members>Schedulable__c.Frequency__c</members>
+        <members>Schedulable__c.Last_Time_Run__c</members>
+        <members>Trigger_Handler__c.Active__c</members>
+        <members>Trigger_Handler__c.Asynchronous__c</members>
+        <members>Trigger_Handler__c.Class__c</members>
+        <members>Trigger_Handler__c.Load_Order__c</members>
+        <members>Trigger_Handler__c.Object__c</members>
+        <members>Trigger_Handler__c.Trigger_Action__c</members>
+        <members>Trigger_Handler__c.User_Managed__c</members>
+        <members>Trigger_Handler__c.Usernames_to_Exclude__c</members>
+        <members>Filter_Group__mdt.Description__c</members>
```

Comment:
@bethbrains This is definitely not in alphabetical order (F does not come after T).